### PR TITLE
Ported the configuration to libconfig format

### PR DESCRIPTION
**Pull request by MishimaHaruna**
Based on Panikon's work, fixed, improved and updated for modern Hercules.
Includes general improvements as well.
See the individual commits for details.

---
This change is [Reviewable](https://reviewable.io/reviews/herculesws/hercules/1399)

### CODE REVIEW

**Comment by EPuncker** [comment]
that's neat

File: npc/pre-re/scripts.conf [hunk 1]
```diff
@@ -29,84 +29,84 @@
 //=========================================================================
 
 //================= Airport ===============================================
-npc: npc/pre-re/airports/izlude.txt
+"npc/pre-re/airports/izlude.txt",
```
Comment (kyeme) [inline npc/pre-re/scripts.conf]: What is the format now for delnpc? :kissing:

File: npc/scripts_removed.conf [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+//= along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//=========================================================================
+//= Hercules global npc removed list
+//=========================================================================
+// All NPCs that you wish to remove from your loading list should be put in
+// this file
+//=========================================================================
+
+// If instead of a path to an NPC is put "all" no file will be loaded
+npc_removed_list: (
```
Comment (Ridley8819) [inline npc/scripts_removed.conf]: <3

File: npc/pre-re/scripts.conf (hunk 1 above)
Comment (MishimaHaruna) [inline npc/pre-re/scripts.conf]: Yup, that's right :)

File: src/char/char.c [hunk 3]
```diff
@@ -136,7 +138,16 @@ int max_connect_user = -1;
 int gm_allow_group = -1;
 int autosave_interval = DEFAULT_AUTOSAVE_INTERVAL;
 int start_zeny = 0;
-int start_items[MAX_START_ITEMS*3];
+
+/// Start items for new characters
+struct start_item_s {
```
Comment (4144) [inline src/char/char.c]: Probably add also cards into start items?

File: npc/pre-re/scripts.conf (hunk 1 above)
Comment (kyeme) [inline npc/pre-re/scripts.conf]: @MishimaHaruna delnpc documentation should be updated too :D

**Pull request merged**